server.ts

```diff
@@ -1,12 +1,27 @@
 // server.ts - Imports the app and sets it to listen on the indicated port.
 
+// Import the app from src
 import app from "./src/app";
 
+// Import morgan library for logging http requests
+import morgan from "morgan";
+
+// Read port from environment variables
 const PORT = process.env.PORT;
 
+// Read debugging mode from environment variables
+const DEBUG = process.env.DEBUG === "true" || false;
```

Comment:
I'm thinking we should add a section to the `README.md` file listing the required environmental variables in one place since that file isn't version controlled.

Reply:
I've made a Jira issue for this. I don't think it should block this PR.

Reply:
I have assigned this to myself on Jira. I will work on it after the merge since the logging branch is missing changes to the README that are present in main.